### PR TITLE
Replace airbrake with sentry

### PR DESCRIPTION
**Pull request by h-lame**
For: https://trello.com/c/6RdsLgOY/51-update-government-frontend

### CODE REVIEW

**Comment by fofr** [comment]
Should we include the equivalent of this too:
https://github.com/alphagov/collections/pull/383

**Comment by fofr** [comment]
Can I suggest `ContentParts` rather than `PartsPresenterMethods`, in keeping with other modules. Maybe a comment alluding to clash too? How did this error present itself?

File: app/controllers/content_items_controller.rb [hunk 1]
```diff
@@ -30,7 +28,7 @@ def set_guide_draft_access_token
   end
 
   def load_content_item
-    content_item = content_store.content_item(content_item_path)
+    content_item = Services.content_store.content_item(content_item_path)
```
Comment (fofr) [inline app/controllers/content_items_controller.rb]: :+1: :100:

File: app/presenters/guide_presenter.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def page_title
 
   def has_parts?
     unless parts.any?
-      Airbrake.notify("Guide with no parts",
+      GovukError.notify("Guide with no parts",
         error_message: "Guide rendered without any parts at #{base_path}")
```
Comment (tijmenb) [inline app/presenters/guide_presenter.rb]: This needs wrapping in `extra: {}`

Comment (h-lame) [reply app/presenters/guide_presenter.rb]: Oh, didn't know that, will do.

**Comment by h-lame** [comment]
@fofr - the error presented itself in all the tests for the routes and controllers that included the module failing with method not found errors.  I see what you mean about the names of the other modules, `PartsPresenterMethods` does stick out like a sore thumb.  Looking at them though I'm surprised we don't have the same problem with them as some have fairly generic names (`Body` and `Metadata` in particular).  Perhaps it makes sense to introduce a namespace for them to make sure they're unique to us.  Something like `ContentItemAttributes::` maybe?

**Comment by fofr** [comment]
@h-lame I agree, we should namespace in a later PR. Wonder why the Faraday module isn't namespaced.

**Comment by h-lame** [comment]
@fofr - looks like there's no github hosted ruby code with a class called `ContentParts` so I'll use that for now (see: https://github.com/search?l=ruby&q=ContentParts&type=Code&utf8=%E2%9C%93).  Will do a namespace in a later PR.

File: test/presenters/parts_presenter_methods_test.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-module PartsStubs
```
Comment (fofr) [inline test/presenters/parts_presenter_methods_test.rb]: Filename also needs updating

Comment (h-lame) [reply test/presenters/parts_presenter_methods_test.rb]: Oops! Just spotted this myself; sorry!

**Pull request merged**